code/python/example.py

```diff
@@ -1,21 +1,24 @@
-# pip3 install neo4j-driver
-# python3 example.py
+# (requires Python 3.6 or newer and pip, check with `pip --version`)
+# pip install neo4j
+# python example.py
 
-from neo4j import GraphDatabase, basic_auth
+from neo4j import GraphDatabase
 
-driver = GraphDatabase.driver(
-  "neo4j+s://demo.neo4jlabs.com:7687",
-  auth=basic_auth("mUser", "s3cr3t"))
+def products_of_category(transaction, movie_title_):
+    cypher_query = (
+        "MATCH (m:Movie {title:$movieTitle})<-[:ACTED_IN]-(a:Person)" 
```

Comment:
no double quotes but triple quote strings please otherwise this breaks the templating and is also not idiomatic